conf.py

```diff
@@ -13,13 +13,21 @@
 # Sphinx extension module names and templates location
 sys.path.append(os.path.abspath("_extensions"))
 extensions = [
-    "gdscript",
-    "godot_descriptions",
-    "sphinx_search.extension",
     "sphinx_tabs.tabs",
     "sphinx.ext.imgmath",
 ]
 
+sphinx_tabs_nowarn = True
+
+if not os.getenv("SPHINX_NO_GDSCRIPT"):
```

Comment:
Since the use case is only for CI, maybe this can all be grouped in a single env variable like `GODOT_LEAN_BUILD`?

Reply:
I've thought about that, but I think I'd like to be able to disable building the search index which I never use locally, and still be able to have GDScript lexing work and see the pages locally as they would be on RTD.

I don't care strongly however, I can change this to GODOT_LEAN_BUILD too :)